docker/conda/environments/cuda11.8_examples.yml

```diff
@@ -35,6 +35,10 @@ dependencies:
     - s3fs>=2023.6
     - pip
     - wrapt=1.14.1 # ver 1.15 breaks the keras model used by the gnn_fraud_detection_pipeline
+    - dgl==1.0.2+cu118
+    - torchmetrics==0.11.4
```

Comment:
Is `torchmetrics` a hard requirement or a nice-to-have?
I see it being used in `models/training-tuning-scripts/fraud-detection-models/training.py` but that appears to have its own requirements file.

Reply:
Yes, `torchmetrics` could be removed from this yaml file. It is not used in the pipeline except for training.